SUMMARY.md

```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
+  * [Neo4j Database](backend/neo4j-database.md)
```

Comment:
This whole section `neo4j` will be moved to a top level folder in https://github.com/Human-Connection/Human-Connection/pull/485: 
```suggestion
  * [Neo4j Database](neo4j/README.md)
```